README.md

```diff
@@ -8,12 +8,20 @@ This is fully inspired by the following blog post: https://simonwillison.net/202
 
 ## Usage
 
-You can start [here](https://github.com/Arcadia-Science/nextflow-template-repository/generate) and follow the prompts. Alternatively, you can click on the "Use this template" button on this repository and follow the prompts.
+You can click on the "Use this template" button on this repository and follow the prompts. Alternatively, you can start [here](https://github.com/Arcadia-Science/nextflow-template-repository/generate) and follow the prompts. Please follow these instructions:
+
+- The name of your repository will be the name of the Nextflow pipeline.
+- Add a one-line description of your pipeline.
+- Make sure to check "Include all branches". This will copy the `TEMPLATE` branch that's used by `nf-core` to keep modules up-to-date and make following the `nf-core` guidelines easier.
```

Comment:
To explain my confusion on this point -- this repo doesn't seem to have a `TEMPLATE` branch afaict

Reply:
Yup, I am aware, that'll be added once this PR is merged. That was an oversight on our end